engine/runtime/src/main/java/org/enso/interpreter/runtime/TruffleCompilerContext.java

```diff
@@ -729,6 +728,11 @@ public Source getSource() throws IOException {
       return module.getSource();
     }
 
+    @Override
+    public CharSequence getCharacters() throws IOException {
+      return module.getSource().getCharacters();
```

Comment:
Is a `NullPointerException` possible here? I thought that `module.getSource()` can return `null`.